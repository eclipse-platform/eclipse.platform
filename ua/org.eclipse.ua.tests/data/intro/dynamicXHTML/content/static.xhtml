<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<!--
   A static XHTML page for sanity test. No dynamic content here.
-->

<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>static.xhtml</title>
</head>

<body>
<h2>static.xhtml</h2>
<p>
This page contains static XHTML content only.
</p>
</body>
</html>
