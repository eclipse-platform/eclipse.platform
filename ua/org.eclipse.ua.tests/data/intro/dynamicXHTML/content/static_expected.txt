<!--
   A static XHTML page for sanity test. No dynamic content here.
--><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>

<title>static.xhtml</title>
</head>

<body>
<h2>static.xhtml</h2>
<p>
This page contains static XHTML content only.
</p>
</body>
</html>